Microsoft.Kiota.Abstractions.Tests/QueryParametersBaseTests.cs

```diff
@@ -13,7 +13,7 @@ public void SetsSelectQueryParameters()
             var requestInfo = new RequestInformation
             {
                 HttpMethod = Method.GET,
-                UrlTemplate = "http://localhost/me?select={select}"
+                UrlTemplate = "http://localhost/me?select={%24select}"
```

Comment:
```suggestion
                UrlTemplate = "http://localhost/me?%24select={select}"
```

Reply:
```suggestion
                UrlTemplate = "http://localhost/me{?%24select}"
```
actually this syntax makes the parameter mandatory, which is not what we want.